br/pkg/lightning/backend/local/local.go

```diff
@@ -1789,14 +1789,18 @@ func (local *local) isIngestRetryable(
 		if newRegion != nil {
 			retryTy = retryWrite
 		}
-		return retryTy, newRegion, errors.Errorf("epoch not match: %s", errPb.GetMessage())
+		return retryTy, newRegion, errors.Annotate(common.ErrKVEpochNotMatch, errPb.GetMessage())
```

Comment:
It's better to use `common.ErrKVEpochNotMatch.GenWithStackXXX` instead of `errors.Anotate` for better readability. See https://github.com/pingcap/tidb/issues/32133 for more details.

Reply:
`common.ErrKVEpochNotMatch.GenWithStackXXX` will clone the current error, then we cannot check it using the defined error in `IsRetryableError`.

Reply:
Use this function to check.
https://github.com/pingcap/tidb/blob/db07c2e0c80d745bdf3030826bdad7fd29da8f9e/br/pkg/errors/errors.go#L10